### PR TITLE
Add missing autoloads

### PR DESCRIPTION
**Pull request by Silex**
Hello,

This should be self-explanatory, ask if there's any questions.

### CODE REVIEW

File: crux.el [hunk 1]
```diff
@@ -412,6 +412,7 @@ buffer is not visiting a file."
 
 Meant to be used as `find-file-hook'.
 See also `crux-reopen-as-root-mode'."
+  (interactive)
```
Comment (bbatsov) [inline crux.el]: That function is not meant to be used interactively. There's `crux-sudo-edit` for this purpose.

Comment (Silex) [reply crux.el]: Ah, ok I'll remove the commit

**Comment by Silex** [comment]
Okay I removed the useless commit.

**Comment by bbatsov** [comment]
👍

**Pull request merged**